packages/tools/tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "target": "esnext",
     "experimentalDecorators": true,
     "strict": true,
+    "esModuleInterop": true,
```

Comment:
do we need this?

Reply:
Yes, because once one of your dependencies uses it - you must use it. That's why that lot of changes, migrated to be `esModulesInterop` compatible.

Reply:
Which one?

Reply:
one of `@storybook/*`, obviously, nothing else did change :)